plugins/org.eclipse.n4js/src/org/eclipse/n4js/validation/validators/N4JSXValidator.xtend

```diff
@@ -147,6 +147,9 @@ class N4JSXValidator extends AbstractN4JSDeclarativeValidator {
 	 */
 	@Check
 	def public void checkReactElementBinding(JSXElement jsxElem) {
+		if (jsxElem.jsxElementName === null) {
+			return;
+		}
```

Comment:
why is this required, now?

Reply:
Removed. Relicts of attempt to make jsxElementName optional...